docs/UserGuide.md

```diff
@@ -353,16 +347,24 @@ end time `15:00` and location `Meeting room A`.
 Edits an existing meeting in your displayed meeting list.
 
 Format: `edit INDEX [t/TITLE] [from/FROM] [to/TO] [c/CONTACTS] [l/LOCATION] [d/DESCRIPTION]`
-* `INDEX` is the index number of the meeting in your displayed meeting list.
-* `INDEX` must be a positive integer.
-* At least one of the fields of the meeting must be changed.
-* The existing values of the specified meeting will be updated to the input values.
-* You can clear an optional field by inputting an empty parameter, e.g. `edit 1 d/` will empty the description of the 1st meeting.
+
+Parameter      | Description                   | Example 
+---------------|-------------------------------|--------------------------------
+`INDEX`        | The index number of the meeting to be edited in your displayed deliverable list. <br><br>Details:<br>It must be a positive integer. |`1`: represents deliverable with index 1 in the list
```

Comment:
I feel that the table wise can follow the format of deliverable and just have the index param. But the notes below is good, we should have it for editing deliverable as well.